bom/forms.py

```diff
@@ -561,6 +561,8 @@ def clean(self):
 
                 skip = False
                 part_revision = PartRevision()
+                # Get the imported revision
```

Comment:
Thanks @ks156. Sorry if this is picky but do you mind removing the comment? The change is clear enough!

Reply:
I'll just take care of it to merge this in.

Reply:
Thank you for having removed this comment yourself 😄